host/sgx/create.c

```diff
@@ -562,6 +572,104 @@ oe_result_t oe_sgx_validate_enclave_properties(
     return result;
 }
 
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+static oe_result_t _eeid_resign(
+    oe_sgx_load_context_t* context,
+    oe_sgx_enclave_properties_t* properties)
+{
+    oe_result_t result = OE_OK;
+    oe_eeid_t* eeid = context->eeid;
+
+    if (eeid && eeid->data_size > 0)
+    {
+        sgx_sigstruct_t* sigstruct = (sgx_sigstruct_t*)properties->sigstruct;
+
+        OE_SHA256 ext_mrenclave;
+        oe_sha256_final(&context->hash_context, &ext_mrenclave);
+
+        OE_CHECK(oe_sgx_sign_enclave(
+            &ext_mrenclave,
+            properties->config.attributes,
+            properties->config.product_id,
+            properties->config.security_version,
+            OE_DEBUG_SIGN_KEY,
```

Comment:
For the record, I'm still against the use of the `OE_DEBUG_SIGN_KEY` in this context; it's fine as a convenience for an experimental feature, but in a production context there need to be guards against the use of MRSIGNER (e.g. in `oe_get_seal_key()`), or this needs to be something that the user specifies rather than an insecure-by-default value provided by the SDK.

Reply:
It's not particularly hard to add, but I still think it's unnecessary. I'm not a security expert though, so I may well be missing something that's obvious to you. @ad-l do you have more comments on this?

Reply:
An almost as easy way around this would be to use a randomly generated throw-away key every time.

If you aren't going to verify the signature anyway it doesn't really matter what key. This prevents verifiers or other applications from accidentally taking a dependency on an insecure key. In the case the key is very secure but doesn't need to be managed in any way.

Reply:
That's right, a random key would probably provide the same security. The difference is that it has to be distributed to everybody who needs to know it after enclave creation. In multi-party settings that may be a complication as the parties need to agree on a key, or decide on a trustworthy source.